src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -1463,13 +1463,26 @@ void scrollElementToFirstCellAndSendKeys(WebElement spreadsheetElement, String v
         private static final String START_OCCURRED_ATTRIBUTE = "__ajaxStartOccurred__";
 
         /**
-         * Registers `ajaxStart` and `ajaxStop` handlers to track the state of an AJAX request.
+         * Waits for all jQuery AJAX requests to complete.
+         */
+        void waitForAllRequestsComplete() {
```

Comment:
On second thought, maybe `waitForNoActiveRequests` is more accurate since it will make more sense for the case when there is no current ongoing request.